draft-regext-rdap-extensions.md

```diff
@@ -140,6 +183,37 @@ extensions that may have an "author" structure. But the JSON contained
 within "lunarNIC_author" need not be prepended as the extension collision
 is avoided by "lunarNIC_author".
 
+## Suffixless Prefixes #{suffixless_prefixes}
```

Comment:
'Suffixless prefixes' is an expression that might be confusing for some readers.  Suggested:

```
## Use of bare extension identifier

Some RDAP extensions use the extension identifer itself (i.e. without appending some other name to it) as a URL path segment, query parameter, or JSON name.  Consider the example in ... Using the "bare extension identifier" pattern ...

{ ... }

The use of a "bare extension identifier" ...
```

(Another option here is 'null suffix', but I think 'bare extension identifier' works better.)

Reply:
This works for me.